src/main/resources/ilex/resources/gui_en_US.properties

```diff
@@ -24,6 +24,7 @@ EditorMenuSet.cut = Cut
 EditorMenuSet.copy = Copy
 EditorMenuSet.paste = Paste
 EditorMenuSet.delete = Delete
+EditorMenuSet.checkForNew = Check for New
```

Comment:
need Spanish translation for  this.  If we use 'Update' or 'Refresh'   Spanish could be 'Actualizar'